src/Generation/GapicClientV2Generator.php

```diff
@@ -723,6 +724,16 @@ private function construct(): PhpClassMember
                             'A callable which returns the client cert as a string. This can be used to provide',
                             'a certificate and private key to the transport layer for mTLS.'
                         )
+                    ),
+                    PhpDoc::type(
+                        Vector::new([
+                            $ctx->type(Type::false()),
+                            $ctx->type(Type::fromName(LoggerInterface::class))
+                        ]),
+                        'logger',
+                        PhpDoc::text(
+                            'A PSR-3 compliant logger. If set to false logging is disabled ignoring the \'OOGLE_SDK_DEBUG_LOGGING\' flag'
```

Comment:
Typo in the env name!
```suggestion
                            'A PSR-3 compliant logger. If set to false logging is disabled ignoring the \'OOGLE_SDK_DEBUG_LOGGING\' flag'
```